packages/ui-common/src/ContextGate.tsx

```diff
@@ -42,6 +43,8 @@ const DISCONNECTED_STATE_PROPERTIES = {
   }
 };
 
+const url = process.env.WS_URL || settings.apiUrl || undefined;
```

Comment:
Let's remove `process.env.WS_URL` for now. It's an unneeded feature, if it's required then we can add it back, maybe via a --wsUrl flag.